frontend/src/atom/textAtom.ts

```diff
@@ -4,15 +4,15 @@ import { atom, atomFamily } from 'recoil';
 export const TEXT_WHITE: RGBColor = { r: 251, g: 251, b: 251, a: 1 };
 export const TEXT_BLACK: RGBColor = { r: 10, g: 10, b: 10, a: 1 };
 
-export const TEXT_LG = 'text-lg';
-export const TEXT_MD = 'text-md2';
-export const TEXT_SM = 'text-sm';
-export type TextSizeType = 'text-lg' | 'text-md1' | 'text-md2' | 'text-sm' | 'text-xsm';
+export const TEXT_LG = 'tablet:text-[34px] text-[21px]';
+export const TEXT_MD = 'tablet:text-[26px] text-[16px]';
+export const TEXT_SM = 'tablet:text-[22px] text-[14px]';
+export type TextSizeType = string;
```

Comment:
사실 string도 any와 비슷한 상태가 되버립니다.
```suggestion
export type TextSizeType = 'tablet:text-[34px] text-[21px]' | 'tablet:text-[26px] text-[16px]' | 'tablet:text-[22px] text-[14px]' ;
```
이렇게 고정된 값으로 있으면 정해진 문자열 만 들어올 수 있고, TEXT_LG 값을 대입할 수 있을 거에요.

Reply:
제안 미반영하고 resolve 하셔도 됩니다!

Reply:
좋은 방식인 것 같습니다.
아무래도 type을 string으로 정의하는건 안좋은 코드라고 생각합니다.

이부분 실제 적용해보고 잘 동작하는 것 확인하였습니다.

제안해주신대로 적용 후 머지하도록 하겠습니다!